ocs_ci/templates/workloads/cephfs_stress/cephfs_filesystem_monitor.py

```
@@ -0,0 +1,633 @@
+    logger.info(f"All monitoring threads started ({len(threads)} threads)")
+    logger.info("Monitoring is now active. Press Ctrl+C to stop.")
+    
+    try:
+        # Keep main thread alive
+        while not shutdown_event.is_set():
+            time.sleep(10)
+            
+            # Check if any thread has died unexpectedly
+            for t in threads:
+                if not t.is_alive():
+                    logger.error(f"Thread {t.name} has died unexpectedly!")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)